docs/en/getting-started/install.md

```diff
@@ -9,7 +9,7 @@ slug: /en/install
 You have two options for getting up and running with ClickHouse:
 
 - **[ClickHouse Cloud](https://clickhouse.com/cloud/):** the official ClickHouse as a service, - built by, maintained, and supported by the creators of ClickHouse
-- **[Self-managed ClickHouse](https://github.com/ClickHouse/ClickHouse):** ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture
+- **[Self-managed ClickHouse](#self-managed-install):** ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture
```

Comment:
```suggestion
- **[Self-managed ClickHouse](#self-managed-requirements):** ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture
```

Reply:
I think that you are right regarding that the actual instructions that the reader needs are below the fold.  I would suggest linking to the system requirements, as for readers who are planning to do more than a quick play should see the requirements.

If you agree then just hit commit under this message and it will update the URL.